src/components/TweetView/Tweet.scss

```diff
@@ -0,0 +1,36 @@
+.block {
+  border-bottom: 1px solid #e1e8ed;
+  border-left: 1px solid #e1e8ed;
+  border-right: 1px solid #e1e8ed;
+  padding: 10px 15px;
+  list-style: none;
+  background-color: white;
+  &:hover {
+    background-color: #f5f8fa;
```

Comment:
@victoriasloan is working on this, will come through in a separate PR.

@girwin575 Can you make the quick imageUrl/alt/React. changes and then we'll merge this? Means @victoriasloan can base the variables update off master then, which will have the 2 current View PRs merged, so all CSS is covered.